src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -5,7 +5,7 @@
 
 <template>
 	<!-- Message Actions -->
-	<div v-click-outside="handleClickOutside">
+	<div v-click-outside="handleClickOutside" :class="$props['class']">
```

Comment:
can't be just `:class="class"` ?

Reply:
No, because `class` is reserved work in JavaScript, e.g., you cannot create a variable called `class` or `function`. But you can create an object property with any name.